best-practices/tagging/index.html

```diff
@@ -300,8 +300,8 @@ <h4>Headings</h4>
 				<aside class="example" title="Simple heading">
 					<pre><code class="html">&lt;h1>⠠⠊⠝⠞⠗⠕⠙⠥⠉⠞⠊⠕⠝&lt;/h1></code></pre>
 				</aside>
-
 				<aside class="example" title="Multiline heading">
+					<p>Note that while <code>&lt;br/></code> can be used to force multiple lines, doing so will hardcode those breaks. It is better to identify the heading with either a tag or class and use CSS to center the braille &lpar;<code>text-wrap: balance</code>&rpar;. See <a href=https://daisy.github.io/ebraille/best-practices/styling/#line-breaking>Styling Best Practices, section 2.7</a>.</p>
```

Comment:
I'm not sure what the best practice is when referencing other sections. E.g. should we include the number (because it could change)?

Reply:
No, don't use the section number. You can use just the section title, as they're less likely to change, but if you can write a descriptive text that doesn't rely on either that's even better (e.g., "for more information, refer to the the section on ... [brl-style]").